app/src/main/java/com/example/ide/ui/screen/ChatScreen.kt

```
@@ -1,524 +1,524 @@
+                                labelColor = MaterialTheme.colorScheme.onSecondaryContainer
+                            )
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun ChatMessageItem(
```

detekt (1.23.8)

[warning] The function ChatMessageItem appears to be too complex based on Cyclomatic Complexity (complexity: 37). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)